src/test/java/com/conveyal/datatools/manager/controllers/api/UserControllerTest.java

```diff
@@ -42,34 +40,32 @@
  * certain responses needed to verify functionality.
  */
 public class UserControllerTest extends UnitTest {
-    private String emailForExistingAccount = "test-existing-user@test.com";
-    private ObjectMapper mapper = new ObjectMapper();
+    private final String emailForExistingAccount = "test-existing-user@test.com";
+    private final ObjectMapper mapper = new ObjectMapper();
     private final String USER_SEARCH_API_PATH = USERS_API_PATH + "?sort=email%3A1&per_page=10&page=0&include_totals=false&search_engine=v3&q=email%3A";
-    /**
-     * This sets up a mock server that accepts requests and sends predefined responses to mock an Auth0 server.
```

Comment:
This comment (which I think is nice to have) was lost.

Reply:
Updated.